tests/integration/api/notice_error/test_good_4_args_error_handler.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should record a traced error when newrelic_notice_error is
+used as a callback handler for set_error_handler().
+*/
+
+/*EXPECT_TRACED_ERRORS
+[
+  "?? agent run id",
+  [
+    [
+      "?? when",
+      "OtherTransaction/php__FILE__",
+      "4 arg error",
+      "NoticedError",
+      {
+        "stack_trace": [
+          " in newrelic_notice_error called at ? (?)",
+          " in trigger_error called at __FILE__ (??)",
+          " in a called at __FILE__ (??)"
+        ],
+        "agentAttributes": "??",
+        "intrinsics": "??"
+      },
+      "?? transaction ID"
+    ]
+  ]
+]
+*/
+
+/*EXPECT_ERROR_EVENTS
+[
+  "?? agent run id",
+  {
+    "reservoir_size": "??",
+    "events_seen": 1
+  },
+  [
+    [
+      {
+        "type": "TransactionError",
+        "timestamp": "??",
+        "error.class": "NoticedError",
+        "error.message": "4 arg error",
+        "transactionName": "OtherTransaction\/php__FILE__",
+        "duration": "??",
+        "nr.transactionGuid": "??",
+        "guid": "??",
+        "sampled": true,
+        "priority": "??",
+        "traceId": "??",
+        "spanId": "??"
+      },
+      {},
+      {}
+    ]
+  ]
+]
+*/
+
+/*EXPECT_SPAN_EVENTS_LIKE
```

Comment:
Might make sense to add an EXPECT_ANALYTICS_EVENTS block to verify the attributes were added there as well.